github/respository_rules_utils.go

```diff
@@ -300,6 +300,22 @@ func expandRules(input []interface{}, org bool) []*github.RepositoryRule {
 		rulesSlice = append(rulesSlice, github.NewPullRequestRule(params))
 	}
 
+	// Pull request rule
```

Comment:
nit: incorrect comment, left over from copy and paste?

Reply:
Yup, thanks.